cli/azd/CHANGELOG.md

```diff
@@ -1,15 +1,40 @@
 # Release History
 
-## 1.8.0-beta.1 (Unreleased)
+## 1.8.0 (2024-04-09)
 
 ### Features Added
 
+- [[3569]](https://github.com/Azure/azure-dev/pull/3569) Adds `--from-code ` flag to initialize from existing code when running `azd init`
+- Dotnet Aspire:
+  - [[3364]](https://github.com/Azure/azure-dev/pull/3364) Supports prompting behavior to an external host
+  - [[3612]](https://github.com/Azure/azure-dev/pull/3612) Supports Aspire apps with multiple exposed ports
+  - [[3484]](https://github.com/Azure/azure-dev/pull/3484) Discovers export port from the result of `dotnet publish`
+  - [[3560]](https://github.com/Azure/azure-dev/pull/3560) VS-Server: Implements `DeleteEnvironmentAsync` RPC for Aspire Server
+  - [[3556]](https://github.com/Azure/azure-dev/pull/3556) Adds Aspire volumes support
+  - [[3561]](https://github.com/Azure/azure-dev/pull/3561) Supports more input generation in Aspire manifest
+
 ### Breaking Changes
 
+- [[3589]](https://github.com/Azure/azure-dev/pull/3589) Adds secure() for secrets in container apps. The `container-app.bicep` and `container-app-upsert.bicep` modules now mark the secrets parameter as secure(), and expect secrets to be an object (versus an array). You will need to reformat your array of [{name: 'secret-name', value: secretValue}, ...] as an object like {'secret-name': secretValue, ...} instead. Thanks @pamelafox for the contribution
```

Comment:
Can we shorten this? ex: Secrets are now marked as secure() in `container-app.bicep` and `container-app-upsert.bicep`